app/models/user.rb

```diff
@@ -23,29 +23,28 @@
 class User < ApplicationRecord
   mount_uploader :avatar, AvatarUploader
 
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+
+  acts_as_liker
   acts_as_followable
   acts_as_follower
+
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-
   validates_integrity_of  :avatar
   validates_processing_of :avatar
-
-  ROLES = [:user, :admin]
-
-  has_many :posts, dependent: :destroy
-  has_many :comments, dependent: :destroy
-  enum role: ROLES
-
   validates :username,
     presence: true,
     uniqueness: { case_sensitive: false },
     length: { minimum: 3 }
 
+  ROLES = [:user, :admin]
```

Comment:
please move constants to the top of page
and you can use for array of symbols this syntax `%i(user admin)`